source/includes/hadoop-eol-spark.rst

```diff
@@ -0,0 +1,5 @@
+.. note::
+
+   The MongoDB Connector for Hadoop is no longer supported and has been
+   succeeded by the
+   `MongoDB Connector for Spark <https://docs.mongodb.com/spark-connector/current/>`__.
```

Comment:
I would recommend something like this that provides a title that calls out to the reader more:

![image](https://user-images.githubusercontent.com/706219/46116264-cb178d00-c1c0-11e8-9684-0f0d462737d7.png)

Maybe:

```
.. admonition:: MongoDB Connector for Spark replaces MongoDB Connector for Hadoop
   :class: important

   If you would like to access MongoDB databases using the Apache Spark libraries, use
   `MongoDB Connector for Spark <https://docs.mongodb.com/spark-connector/current/>`__. The MongoDB
   Connector for Hadoop is no longer supported.
```
   

```

Reply:
Good suggestion, tweaked it a little bit.